### DISCUSSION TITLE
Aggregatorv3Interface problem

### DISCUSSION THREAD

**Discussion by Rahul251952** [Cyfrin/foundry-full-course-cu#2995] [General]
I am facing issue in import
![Screenshot 2024-12-06 000435](https://github.com/user-attachments/assets/3def7336-59af-49dd-ae35-e675377a8de3)
and the error is Unable to resolve imports:
      "./PriceConverter.sol" in "/home/rahul/Hello-foundry/src/Fundme.sol"
      "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol" in "/home/rahul/Hello-foundry/src/Fundme.sol"
with remappings:
[⠊] Unable to resolve imports:ello-foundry/lib/forge-std/src/
      "./PriceConverter.sol" in "/home/rahul/Hello-foundry/src/Fundme.sol"
      "@chainlink/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol" in "/home/rahul/Hello-foundry/src/Fundme.sol"
with remappings:
      forge-std/=/home/rahul/Hello-foundry/lib/forge-std/src/
[⠒] Compiling 30 files with Solc 0.8.19
[⠒] Installing Solc version 0.8.19
[⠢] Failed to install Solc 0.8.19: error sending request for url (https://binaries.soliditylang.org/linux-amd64/list.json)
Error: error sending request for url (https://binaries.soliditylang.org/linux-amd64/list.json)

Context:
- Error #0: client error (Connect)
- Error #1: dns error: failed to lookup address information: Temporary failure in name resolution
- Error #2: failed to lookup address information:
-

**Comment by LordAdhaar** [comment]
Make sure you have updated your foundry.toml with this 

`remappings = ["@chainlink/contracts/=lib/chainlink-brownie-contracts/contracts"]`

**Reply by EngrPips** [reply]
Great heads-up

**Reply by Rahul251952** [reply]
Not working. Still getting the same error.

**Reply by EngrPips** [reply]
Have you actually installed the chainlink library?

**Reply by Rahul251952** [reply]
yes

**Reply by EngrPips** [reply]
You can use `command + p` [for Macbook] or `cltr + p` [for windows] and input the name of the file you want to import and like that you will seethe path to the file, and you can use that native path if remapping is causing you headache

**Reply by Rahul251952** [reply]
no result found. What should I do.

**Reply by EngrPips** [reply]
Run this `forge install smartcontractkit/chainlink-brownie/contract --no commit` then retry

**Reply by Rahul251952** [reply]
Nothing change. I am facing this error for 2 days. Please give me some suggestion.

**Reply by EngrPips** [reply]
> Run this `forge install smartcontractkit/chainlink-brownie/contract --no commit` then retry

Please run this and show us what the output is, You can't share very little or no information and expect us to just magically know what is wrong on your end.

**Reply by Rahul251952** [reply]
![Screenshot 2024-12-06 222536](https://github.com/user-attachments/assets/214c432e-910e-4ae6-b3e9-257ff5a10775)

**Reply by EngrPips** [reply]
Hey @Rahul251952, please run this `forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit` and retry the process. Please share the output of running the command.

**Reply by Rahul251952** [reply]
Thanks for your support buddy. Now it's completely fine.

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.

**Comment by chainflash** [comment]
send a photo of the contents of the foundry.toml file

**Reply by Rahul251952** [reply]
the error has been corrected. Thanks for reaching out to me.
